frontend/test/__runner__/run_timezone_tests

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -eu
+
+# Test a variety of timezones with/without DST
+# https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+#
+# TZ                    UTC offset   UTC DST offset
+# =================================================
+# Etc/UTC                   +00:00           +00:00
+# Europe/London             +00:00           +01:00
+# America/Phoenix           −07:00           −07:00
+# America/Los_Angeles       −08:00           −07:00
+
+DEFAULT_TIMEZONES="Etc/UTC Europe/London America/Los_Angeles America/Phoenix"
```

Comment:
If this is about trying to trigger timezone problems, I think it can also be interesting to also test with a timezone that's always after UTC, such as `Europe/Paris`, code not handling timezone properly with timezone > UTC can have off than one day errors because

```
new Date('2001/01/01 GMT+02:00').toUTCString()
"Sun, 31 Dec 2000 22:00:00 GMT"
```

 ( https://github.com/metabase/metabase/issues/7910 is one example )